model/search.go

```diff
@@ -11,12 +11,13 @@ type SearchPage struct {
 
 // Search represents all search parameters and response data of the search
 type Search struct {
-	Query      string           `json:"query"`
-	Filter     []string         `json:"filter,omitempty"`
-	Filters    []Filter         `json:"filters"`
-	Sort       Sort             `json:"sort,omitempty"`
-	Pagination model.Pagination `json:"pagination,omitempty"`
-	Response   Response         `json:"response"`
+	Query             string           `json:"query"`
+	ValidationProblem string           `json:"ValidationProblem"`
```

Comment:
I would update this to be:

```suggestion
	ErrorMessage string           `json:"error_message,omitempty"`
```

json fields should be `camel_case` and the omitempty will hide this field in response if it is empty `""`